warp10/src/main/java/io/warp10/script/ext/token/TOKENGEN.java

```diff
@@ -139,10 +136,10 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
     Map<Object, Object> params = (Map<Object, Object>) top;
 
     try {
-      TBase token = tokenFromMap(params);
+      TBase token = tokenFromMap(params, getName(), 0L);
```

Comment:
You should leave the DEFAULT_TTL constant in the class and use it here instead of 0L